src/components/styles/GridBox.css

```diff
@@ -19,16 +21,17 @@
     /* border: 1px solid white; */
 }
 
-
-.grid-box.empty {
-    background-color: #181A1B;
-}
-
 .grid-box.food {
     background-color: orange;
 }
 
 .grid-box.snake {
     background-color: springgreen;
     /* margin: 1px; */
+}
+
+.blur {
```

Comment:
This CSS component name is already used in Game.css please use different name to avoid confusion.